Frontend/ui-common/src/Engineering/Helpers/Utils.js

```diff
@@ -39,12 +39,15 @@ export const reboot = window.JOS.emitEvent("java")("onEvent")(JSON.stringify({ev
 
 export const showSplash = window.JOS.emitEvent("java")("onEvent")(JSON.stringify({event:"show_splash"}))()
 
+export const getISODate = function (date) {
+  return date.getFullYear() + "-" + ('0' + (date.getMonth() + 1)).slice(-2) + "-" + ('0' + date.getDate()).slice(-2) 
+}
```

Comment:
this will basically return YYYY-MM-DD, we already have a function for this which will return based on given pattern in common.js can we use that